### DISCUSSION TITLE
Remove: Pause Auto-Refresh check box

### DISCUSSION THREAD

**Discussion by danappsnocode** [cortexrd/Knack-Toolkit-Library#105] [Q&A]
Is there a way to remove the "Pause Auto-Refresh" checkbox?

I auto-refresh only one page in the app.
When moving to other than a full-sized screen it overlaps Logged In fields.

**Comment by cortexrd** [comment]
Hi Dan,

Yes, I've added a new flag just for you!
See **devPauseAutoRefresh** in [https://github.com/cortexrd/Knack-Toolkit-Library/blob/dev/KTL_Customization.js](url)
Just set it to false.

BTW, note that this flag is only visible to those users having the **Developers** role.

Enjoy,
Norm

**Comment by cortexrd** [comment]
Dan,

You may want to read this section on how to use this file in your app:
[https://github.com/cortexrd/Knack-Toolkit-Library/wiki/Customizing-Features](url)

Also, I've converted this discussion into issue #106 

Norm

**Reply by danappsnocode** [reply]
Thanks Norm.  Loving the Toolkit!

**Reply by cortexrd** [reply]
Very happy to read that!

If the new flag solves your problem, I'd appreciate if you could close the associated issue #106 please.

Norm

**Comment by danappsnocode** [comment]
I signed up to github today.  I don't see how to close an issue.
Google mentions a Close button - don't see it.
Don't mean to be a pain.

Dan

Dan Boultinghouse
APPS*nocode*.*com*

**Comment by cortexrd** [comment]
Hi Dan,

So problem, you're absolutely not a pain - lol :)

But I'm surprised...  I thought others could also close an issue.
So you don't see this button?

![image](https://github.com/cortexrd/Knack-Toolkit-Library/assets/42851359/dc74f1a6-417b-4ff1-87a2-2a273d90256e)

If that's the case, I'll take note.  Maybe you can only close issues that you've created.  We'll next time you create one.

BTW, I always tell KTL users to subscribe to this channel:
[KTL Changelog - please subscribe to see all updates!](https://github.com/cortexrd/Knack-Toolkit-Library/discussions/47)


Norm

**Comment by danappsnocode** [comment]
Hi Norm,
I'll make sure I subscribe to the change log.  Attached is my screen shot -
no Close. Sure you are correct re: I wasn't the author.

Regards, Dan

Dan Boultinghouse
APPS*nocode*.*com*

**Reply by cortexrd** [reply]
I don't see any screenshot, but I believe you!
Issue now closed.
Norm
